puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -60,7 +60,16 @@
           values:
             - fast
     execution-strategy:
-      combination-filter: "!(foreman_branch == 'develop' && ruby == '1.9.3')"
+      combination-filter: >
+        "!(foreman_branch == 'develop' && ruby == '1.9.3') ||
+        !(ruby == '1.9.3' && database == 'sqlite') ||
```

Comment:
This is very hard to read, why use negations? Running combinations where database == 'postgresql' or ruby == '2.3' would simplify it considerably.